### PR TITLE
Ticket 38

### LINKED ISSUE #38: Social Media pages
Building out a few more pages from the social media branch, press, and merchandise. will have to add to the nav bar.

### PR DESCRIPTION
**Pull request by margeschrec**
closes #38 
created merchandise  and press page with css pages
changed social media on navbar to follow us dropdown menu
placed merchandise in about us dropdown menu 
form now has a correct trim function

### CODE REVIEW

File: Merch.php [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            </p>
+            <p><em>East Aurora</em><br />
+                185 Main Street<br />
+                East Aurora, NY 14052<br />
+                (716) 652-7959
+            </p>
+            <p><em>Clarence</em><br />
+                8326 Main Street<br />
+                Clarence, NY 14221<br />
+                (716) 710-9464
```
Comment (billythakidd04) [inline Merch.php:252]: I would recommend looking into making these a little more interactive, maybe a google maps link or adding a `tel` tag... Just a suggestion

File: Press.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+				East Aurora, NY 14052<br />
+				(716) 652-7959
+			</p>
+			<p><em>Clarence</em><br />
+				8326 Main Street<br />
+				Clarence, NY 14221<br />
+				(716) 710-9464
+			</p>
+		</div>
+	</footer>
```
Comment (billythakidd04) [inline Press.php:117]: also seeing that this is on more than one page, and is fairly static, this should get wrapped into a seperate file.

File: contact.php [hunk 3]
```diff
@@ -68,85 +75,85 @@
...
-				8326 Main Street<br />
-				Clarence, NY 14221<br />
-				(716) 710-9464
-			</p>
-		</div>
-	</footer>
+                if (!empty(trim($_POST["name"]))) {
+                    $name = trim($_POST['name']);
+                } else {
+                    echo $errors['name'] = "<h2> Name is Required</h2>";
```
Comment (billythakidd04) [inline contact.php:81]: I'm not sure you want to echo here... Though maybe you do... We should talk about your intentions with this line.

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Few items, and we should have a conversation about your forms error div...

File: Merch.php (hunk 1 above)
Comment (vricci518) [reply Merch.php:252]: That was what I was thinking of once I incorporated a working navigation header. Incorporate a common footer file that would link the addresses to Google maps.

File: contact.php (hunk 3 above)
Comment (margeschrec) [reply contact.php:81]: This echo demonstrates to the user that a name is required at the top of the form after they hit submit and did not input a valid name , so it is easy to see. Hope that answers your question.

**Review by vricci518 — APPROVED** [review]

**Pull request merged**